CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 - fix clipboard not working in webxdc apps
 - fix `target=_blank` links in html emails don't work #3408
 
+
+### added
+
+- voice messages
```

Comment:
there is already a section for this above, you need to rebase your pr to newest master though, btw you should try to work from a recent master anyway to get less possibility of merge conflicts